src/guides/v2.3/javascript-dev-guide/javascript/requirejs.md

```diff
@@ -62,6 +62,28 @@ paths: {
 }
 ```
 
+Consider the example of overwriting an HTML file in the adminhtml.
+In this example, the `max-length` value of the text-box in the `adminhtml` is altered. The HTML file is located at `vendor/magento/module_ui/view/base/web/templates/form/element/input.html`
+
+-  Create a `requirejs-config.js` file under `app/code/<Vendor_Name>/<Module_Name>/view/base/`
```

Comment:
Now I can see this properly - it would be better if this list were numbered as it is a guide. So please replace - with 1. for the 5 items